config/routes.yaml

```diff
@@ -12,3 +12,7 @@ curl_search:
 curl_search_category:
     path: /books/category={category}
     controller: App\Controller\BookController::searchBooksByCategory
+
+user:
+    path: /user/favortites
+    controller: App\Controller\UserController::displayFavorites
```

Comment:
rename the name to something more clear what it is about (example: user_favorites)
and the path to /user/{id}/favorites

Reply:
done